src/TextObjects/Block.ts

```diff
@@ -42,7 +42,7 @@ export class TextObjectBlock extends TextObject {
         return obj;
     }
 
-    protected findStartRange(document:TextDocument, anchor: Position): Range {
+    public findStartRange(document:TextDocument, anchor: Position): Range {
```

Comment:
I had to make this functions public. I could not simply use the `apply` method of the `TextObject` because I needed the `startRange.start` and/or the `endRange.start` property...